### PR TITLE
Add Description field in Environments API

### PR DESCRIPTION
**Pull request by timofurrer**

### CODE REVIEW

File: environments.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ type EnvironmentsService struct {
 type Environment struct {
 	ID                  int         `json:"id"`
 	Name                string      `json:"name"`
+	Description         string      `json:"description"`
```
Comment (svanharmelen) [inline environments.go:39]: In the docs `Slug` is before `Description` 🙈

**Review by svanharmelen — APPROVED** [review]
Thanks 👍🏻

**Comment by timofurrer** [comment]
@svanharmelen would you mind releasing this?

**Comment by svanharmelen** [comment]
Waiting on two open PRs to be merged, will make a release after that...

**Comment by svanharmelen** [comment]
Just released `v0.111.0`

**Pull request merged**